Assets/Project/Scripts/MovingPlane.cs

```diff
@@ -7,7 +7,7 @@ public class MovingPlane : MonoBehaviour
 
     [SerializeField] private Transform planeOriginLocation;
     [SerializeField] private Transform planeEndLocation;
-    [SerializeField] private float speed;
+    [SerializeField] public float speed;
```

Comment:
public fields don't need a SerializeField tag

Reply:
ah I see I thought serialized was the method for exposing on the UI wether it was public/private